src/main/java/seedu/duke/userinterface/CliMessages.java

```diff
@@ -1,4 +1,10 @@
 package seedu.duke.userinterface;
 
+import seedu.duke.tasks.TaskList;
+
 public class CliMessages {
+    public static void printAddedTaskMessage(TaskList taskList, String title) {
+        System.out.println("Added: " + title);
+        System.out.println(taskList.getNumberOfTasks() + ":" + taskList.getTask(taskList.getNumberOfTasks() - 1));
```

Comment:
The whole thing inside the print statement could be refactored into the TaskList class, as a `toString()` method or something, do you agree?